services/runlog.go

```diff
@@ -1,7 +1,83 @@
 package services
 
-type RunlogManager struct{}
+import (
+	"context"
+	"sync"
+
+	"github.com/smartcontractkit/external-initiator/blockchain/common"
+	"github.com/smartcontractkit/external-initiator/subscriber"
+
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"go.uber.org/zap"
+)
+
+type Runlog struct {
+	blockchain common.Manager
+
+	quitOnce  sync.Once
+	ctxCancel context.CancelFunc
+
+	chJobTrigger chan subscriber.Event
+	chClose      chan struct{}
+
+	logger *zap.SugaredLogger
+}
+
+func NewRunlog(job string, triggerJobRun chan subscriber.Event, blockchainManager common.Manager) (*Runlog, error) {
+	ctx, cancel := context.WithCancel(context.Background())
+	run := Runlog{
+		blockchain:   blockchainManager,
+		ctxCancel:    cancel,
+		chJobTrigger: triggerJobRun,
+		logger:       logger.Default.With("job", job),
+	}
+	run.logger.Infof("New Runlog job")
+
+	backfilledRequests, err := run.blockchain.Request(common.RunlogBackfill)
+	if err != nil {
+		run.Stop()
+		return nil, err
+	}
+	go run.handleRequest(backfilledRequests.([]interface{})...)
```

Comment:
Maybe we can use a concrete type here for runlog requests instead of interface{}?

Reply:
I agree, and it's something I've already fixed in the next upcoming PR :slightly_smiling_face: